src/actions/controller.ts

```diff
@@ -23,17 +27,49 @@ export class ActionsController {
       }
     }
 
-    public static performScheduledActions() {
-        const actionsSettings = vscode.workspace.getConfiguration();
-        const actionsList: any[] | undefined = actionsSettings.get("actions");
-        if (actionsList && actionsList.length) {
-          for (const action of actionsList) {
-            console.log(
-              `performing action ${action.name} of type ${action.constructor.name}`
-            );
-            _performAction(action);
+    public static async performActionsFromParams() {
+      const logger = getLogger().getChildLogger({label: "performActionsFromParams"});
+      const actionsList = [];
+
+      const actionParam = await getParameter("action");
+      const actionsParam = await getParameter("actions");
+      logger.trace(`configuration - action= ${actionParam}, actions= ${actionsParam}`);
+      const actionIds = actionParam?.split(",") || [];
+      let actionsIds = actionsParam?.split(",") || [];
+      actionsIds = actionsIds.concat(actionIds);
+      actionsIds = uniq(actionsIds);
+      if (actionsIds.length > 0) {
+        for (const actionId of actionsIds) {
+          const action = ActionsController.getAction(actionId);
+          if (action){
+            logger.trace(`action ${actionId} found`, {action});
+            actionsList.push(action);
+          } else {
+            logger.trace(`action ${actionId} not found`);
           }
-          actionsSettings.update("actions", []);
         }
+        ActionsController.performActions(actionsList);
```

Comment:
just call _performAction for each action, no need to create another method

Reply:
done